pkg/storage/tpr/storage_interface_test.go

```diff
@@ -892,6 +894,114 @@ func TestDeleteWithNamespace(t *testing.T) {
 	}
 }
 
+func TestWatch(t *testing.T) {
```

Comment:
I learned the hard way that each test case should be repeated 1. for a resource type that is _not_ namespaced 2. for a resource type that _is_. Things will slip through the cracks if we don't explicitly test both of those scenarios.